SPECS/gd/gd.spec

```diff
@@ -22,7 +22,7 @@ Distribution:   Mariner
 %define lname libgd3
 Name:           gd
 Version:        2.3.0
-Release:        2%{?dist}
+Release:        3%{?dist}
```

Comment:
Also, please lint. Or at least move the `Vendor` and `Distribution` tags to the right place. That's the minimum for all specs migrated from `SPECS-EXTENDED`.